### PR TITLE
Fix for #2100 and #2246:: Using Ringtone class to control ringtones for incoming calls.

### PR DESCRIPTION
**Pull request by lomion0815**
Fix for #2100 and #2246: Using Ringtone and RingtoneManager classes to control ringtones for incoming calls.

Using systemwide setting for ringtone and volume.

### Pull Request Checklist

- [x] Changes has been tested on an Android device or Android emulator with API 21
- UI change has been tested on both light and dark themes => N.A.
- [x] Pull request is based on the develop branch
- [x] Pull request updates [CHANGES.md](https://github.com/vector-im/element-android/blob/develop/CHANGES.md)
- Pull request includes screenshots or videos if containing UI changes => N.A.
- [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)

Signed-off-by: Markus Kastner <@markus:matrix.sarkasti.eu>

### CODE REVIEW

**Comment by lomion0815** [comment]
@BillCarsonFr I would kindly ask you to review this PR!

**Comment by lomion0815** [comment]
Successfully retested based on v1.0.9

File: vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt [hunk 1]
```diff
@@ -150,6 +154,7 @@ class CallAudioManager(
 
     fun onCallConnected(mxCall: MxCall) {
         Timber.v("##VOIP: AudioManager call answered, adjusting current sound device")
+        setupAudioManager(mxCall)
         adjustCurrentSoundDevice(mxCall)
```
Comment (BillCarsonFr) [inline vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt:158]: I think there is no need to call that anymore as it is called at the end of #setupAudioManager()

File: vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt [hunk 2]
```diff
@@ -93,6 +93,10 @@ class CallAudioManager(
 
     fun startForCall(mxCall: MxCall) {
         Timber.v("## VOIP: AudioManager startForCall ${mxCall.callId}")
+    }
+
+    fun setupAudioManager(mxCall: MxCall) {
```
Comment (BillCarsonFr) [inline vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt:98]: make private?

**Review by BillCarsonFr — CHANGES_REQUESTED** [review]
Thanks, for the PR
Just two comments, and will be ok for me to merge

File: vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt (hunk 1 above)
Comment (lomion0815) [reply vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt]: Agreed! Changed in aa2a24e

File: vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt (hunk 2 above)
Comment (lomion0815) [reply vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt]: Made private in aa2a24e

**Comment by lomion0815** [comment]
Changes requested by @BillCarsonFr done.
@bmarty : My branch is based on 1.0.9 and compiles and works. Can you merge it this way or do I have to merge develop/LATEST first?

**Review by BillCarsonFr — APPROVED** [review]
Approved

**Pull request merged**